src/main/java/net/blay09/mods/waystones/block/WaystoneBlock.java

```diff
@@ -91,48 +114,52 @@ public float getPlayerRelativeBlockHardness(BlockState state, PlayerEntity playe
         return super.getPlayerRelativeBlockHardness(state, player, world, pos);
     }
 
-    @Override
-    public boolean isValidPosition(BlockState state, IWorldReader world, BlockPos pos) {
```

Comment:
Why is this removed?

Reply:
This was because the `getStateForPlacement` ended up also preventing placing a waystone directly on top of another, but the `isValidPosition` was preventing it from placing underwater correctly because of the `stateAbove.isAir`